### PR TITLE
Added gun audioclip and screenflash

### PR DESCRIPTION
**Pull request by cnHeider**

### CODE REVIEW

**Comment by Spidersouris** [comment]
add this, please

**Comment by iamchristopher** [comment]
Please review and/ or merge this.

**Comment by nathanchapman** [comment]
@cnHeider There are merge conflicts and the package.json is incorrect. Please make dependency changes to the one in the repo, not the compiled version found in the atom package.

**Comment by cnHeider** [comment]
There you go

**Comment by lexcast** [comment]
@JoelBesada @nathanchapman what do you think on this one? It adds another screen effect, my idea is to keep only one effect for each type (particles, screen, audio) in the core and then make it extensible by other packages.

File: lib/screen-flash.coffee [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  flash: (editorElement) ->
+    normalBG = editorElement.style.background
+    c = @getConfig "color"
+    editorElement.style.background = "rgb(#{c.red},#{c.green},#{c.blue})"
+    setTimeout ->
+      editorElement.style.background = normalBG
+    , @getConfig "duration"
+
+  getConfig: (config) ->
+    atom.config.get "activate-power-mode.screenFlash.#{config}"
```
Comment (lexcast) [inline lib/screen-flash.coffee]: There are some cases when the background changes permanently. Also there is a good variation of this with a `filter: invert(100%)`.
Nevertheless I think for now we will add only the audio effect, I hope the plugin api will be ready soon, so effects (particles, screen and audio) will be interchangeable.

**Comment by cnHeider** [comment]
Thanks, we thinking about dropping that feature anyway. Removed code for screen flashes.

**Comment by lexcast** [comment]
Thank you @cnHeider :tada:

**Pull request merged**